src/eventgrid/azext_eventgrid/_help.py

```diff
@@ -188,7 +202,8 @@
           long-summary: |
             Example: --deadletter-endpoint /subscriptions/{SubID}/resourceGroups/rg1/providers/Microsoft.Storage/storageAccounts/sa1/blobServices/default/containers/containerName
         - name: --endpoint-type
-          short-summary: The type of the destination endpoint.
+          short-summary: The type of the destination endpoint. It is expected that the destination endpoint to be created and available for use before executing any Event Grid command.
```

Comment:
Nit: Consider changing "the destination endpoint to be created" to "the destination endpoint be created"

Reply:
done